source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -15,3 +15,188 @@ Install with Confluent Cloud
+
+      If you have more than 1000 tables to migrate, use the Advanced schema registry tier.
+      Otherwise, the Essentials tier is enough. 
+
+   .. step:: Create a cluster
+
+      Navigate to your :guilabel:`Environment` page to click :guilabel:`Add Cluster`. 
+
+      For most use cases, the Basic and Standard tier clusters are enough. If you 
+      are migrating large volumes of data, consider the Dedicated cluster tier.
+
+   .. step:: Configure and gather permissions and API keys
+
+      To configure Relational Migrator, you must configure and gather the following 
+      permissions and API keys to include them in your ``user.properties`` file:
+
+      The following API keys are found in the Confluent Cloud console. For more 
+      information see, `Use API Keys <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__.
+
+      .. procedure::
+
+         .. step:: Configure Confluent Cloud API keys
+
+            Click the :icon-fa5:`bars` icon on the top right corner, then under 
+            :guilabel:`Administration`, select :guilabel:`Cloud API Keys`.
+
+            Create Confluent Cloud API keys and configure permissions for them. You 
+            have two permission options:
+
+            1. Create global access.
+         
+            2. Create a service account with granular access.
+
+               If you create a service account, you must add the ``Operator`` role for the 
+               environment and the ``CloudClusterAdmin`` role to the cluster. 
+
+            Save these configurations to add to your ``user.properties`` file.
+
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__. 
+
+         .. step:: Configure Kafka cluster API keys
+
+            To access your Kafka cluster, you must configure your Kafka cluster API keys. 
+            Navigate to your :guilabel:`Cluster Overview` and on the left-side navigation 
+            click :guilabel:`API Keys` to create your Kafka cluster API keys.
+
+            Save these configurations to add to your ``user.properties`` file.
+
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__.
+
+   .. step:: Gather more configuration information
+
+      The following information is found in the Confluent Cloud console. For more 
+      information see, `Confluent Cloud Documentation 
+      <https://docs.confluent.io/cloud/current/overview.html>`__.
+
+      .. procedure::
+
+         .. step:: Find and save your environment ID 
+
+            From your :guilabel:`Environments` overview page, choose your 
+            cloud environment, and find your environment ID below the environment name 
+            on the right-side panel. 
```

Comment:
Just a slight consistency tweak, we have been using pane throughout the docs:
```suggestion
            on the right-side pane. 
```

Reply:
Updated.